src/components/Image/Image.tsx

```diff
@@ -0,0 +1,12 @@
+interface ImageProps {
+    onClick?: () => void;
+    src: string;
+    alt: string;
+    className?: string;
+}
+
+const Image = ({ onClick, src, alt, className }: ImageProps) => (
+    <img onClick={onClick} src={src} alt={alt} className={`default-image ${className}`} />
+);
```

Comment:
This will be more readable, and less boilerplate code u need to use
```suggestion
const Image = ({ onClick, src, alt, className, ...restProps }: ComponentProps<"img">) => (
    <img onClick={onClick} src={src} alt={alt} className={`default-image ${className}`} {...restProps} />
);
```